tt-train/sources/ttml/ops/binary_ops.cpp

```diff
@@ -75,7 +75,7 @@ autograd::TensorPtr operator/(const autograd::TensorPtr& a, const autograd::Tens
 
     out->set_value(ttnn::divide(a->get_value(), b->get_value()));
     autograd::GradFunction grad = [a, b, out]() {
-        auto res = ttnn::div_bw(out->get_grad(), a->get_value(), b->get_value(), "None");
+        auto res = ttnn::div_bw(out->get_grad(), a->get_value(), b->get_value(), std::nullopt);
```

Comment:
Do we not default std::nullopt ?

Reply:
yea, that's teh default value. I can remove that